modules/kubernetes-addons/istio/main.tf

```diff
@@ -0,0 +1,138 @@
+resource "kubernetes_namespace" "istio_system" {
+  metadata {
+    name = "istio-system"
+    labels = {
```

Comment:
is this required? are there other labels that we need to allow users to pass in?